data/avgi/avgi.txt

```diff
@@ -0,0 +1,1387 @@
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+phrase "avgi civilian names"
+	phrase
+		"avgi colors" 10
+		"avgi virtues" 5
+		"avgi phases" 5
+	word
+		" "
+	phrase
+		"avgi nature" 3
+		"avgi celestial" 3
+		"avgi elements"
+
+phrase "avgi military names"
+	phrase
+		"avgi virtues"
+	word
+		" "
+	phrase
+		"avgi physics"
+		"avgi elements" 3
+
+phrase "avgi military names"
+	phrase
+		"avgi places"
+
+phrase "avgi fighter names"
+	phrase
+		"avgi letters"
+	word
+		" - "
+	phrase
+		"avgi digits"
+	phrase
+		"avgi digits"
+	phrase
+		"avgi digits"
+
+phrase "avgi digits"
+	word
+		0
+		1
+		2
+		3
+		4
+		5
+
+phrase "avgi letters"
+	word
+		"Alkos"
+		"Bia"
+		"Yenke"
+		"Dia"
+		"Enka"
+		"Tekis"
+		"Isi"
+		"Leto"
+		"Keto"
+		"Minis"
+		"Nunis"
+		"Ski"
+		"Oki"
+		"Pilos"
+		"Rek"
+		"Sek"
+		"Ulos"
+		"Chana"
+
+phrase "avgi colors"
+	word
+		Alabaster
+		Alizarin
+		Aqua
+		Arylide
+		Aureolin
+		Azure
+		Bistre
+		Black
+		Blue
+		Bole
+		Brown
+		Carmine
+		Carnelian
+		Catawba
+		Celadon
+		Cerise
+		Cerul
+		Chartreuse
+		Chiffon
+		Cinereous
+		Citron
+		Claret
+		Coquelicot
+		Cordovan
+		Corsa
+		Drab
+		Fuchsia
+		Fulvous
+		Gainsboro
+		Gamboge
+		Glaucous
+		Gloss
+		Golden
+		Gray
+		Green
+		Grullo
+		Hansa
+		Heliotrope
+		Icterine
+		Indigo
+		Ivory
+		Jade
+		Jasmine
+		Jet
+		Jonquil
+		Keppel
+		Kobe
+		Kobi
+		Kobicha
+		Kombu
+		Lait
+		Lapis
+		Laurel
+		Lazuli
+		Lilac
+		Linen
+		Lisera
+		Magnolia
+		Maize
+		Majorelle
+		Mauve
+		Meringue
+		Mikado
+		Mimi
+		Mindaro
+		Mulberry
+		Nadeshiko
+		Noir
+		Nyanza
+		Ocher
+		Olive
+		Onyx
+		Opera
+		Orange
+		Orchid
+		Palatinate
+		Paolo
+		Paua
+		Payne
+		Periwinkle
+		Persimmon
+		Phlox
+		Phthalo
+		Picotee
+		Pine
+		Pink
+		Prune
+		Puce
+		Purple
+		Purpureus
+		Quinacridone
+		Rajah
+		Razzmatazz
+		Razzmic
+		Red
+		Roje
+		Rosso
+		Ruber
+		Rufous
+		Russet
+		Saffron
+		Sage
+		Satin
+		Schauss
+		Sepia
+		Sienna
+		Silver
+		Sinopia
+		Skobeloff
+		Slate
+		Syracuse
+		Taupe
+		Tenne
+		Thulian
+		Tourmaline
+		Trypan
+		Tyrian
+		Umber
+		Vale
+		Verdant
+		Verdigris
+		Vermilion
+		Vermilion
+		Veron
+		Violet
+		Viridian
+		White
+		Wisteria
+		Xanadu
+		Xanthic
+		Xanthous
+		Yellow
+		Yinmn
+		Zaffre
+
+phrase "avgi places"
+	word
+		Abdera
+		Abila
+		Abydos
+		Acanthus
+		Acharnae
+		Actium
+		Adramyttium
+		Aegae
+		Aegina
+		Aegium
+		Aenea
+		Aenus
```

Comment:
How does one pronounce this?
![image](https://github.com/user-attachments/assets/427447d7-e9e9-453e-bf08-3ee5a4f1c8d0)
😄